src/chapters/installing.xml

```diff
@@ -452,6 +461,31 @@
                                 </informaltable>
                             </entry>
                         </row>
+                        <row>
+                            <entry><link xl:href="https://libwebsockets.org/"
+                                >libwebsockets</link></entry>
```

Comment:
Sorry, this makes me ever so slightly twitchy.  Any reason not to have the closing angle bracket on the line with the rest of that opening tag?

So sorry, just a nitpick...

Reply:
I can move it up. It's just the way that the oXygen editor autoformatted the XML.